website/public/lodging/index.ejs

```diff
@@ -0,0 +1,36 @@
+<h1>Lodging Information</h1>
+
+<h2>STAYING AT SEMIAHMOO RESORT</h2>
+
+<p>UPDATE: Lodging at Semiahmoo is still available for the full CascadiaFest 3-day tickets as well as for single nights from August 2 through August 6.</p>
+
+<p class="text-xs-center"><a href="https://ti.to/event-loop/cascadiafest-2016/" class="btn btn-success">BUY LODGING</a></p>
+
+<h3>STAYING IN BELLINGHAM</h3>
+
+<p>We have arranged for overflow lodging in Bellingham, WA at the Best Western PLUS Heritage Inn and will be running a FREE shuttle to/from the venue. We suggest booking for an August 2 check-in and an August 6 check-out in order to take full advantage of the fun schedule we're working on.</p>
+
+<p><img alt="Best Western" src="/assets/img/bestwestern-map.png" /></p>
+
+<ul>
+  <li>151 East McLeod Road, Bellingham WA, 98226</li>
+  <li>(360) 647-1912</li>
+  <li>http://www.bestwesternheritageinn.com/</li>
+  <li>Rates: Single or Double: $117.99, Triple: $127.99, Quad: $137.99.</li>
```

Comment:
remove: all these listed rates(they changed and I don't think we need to list them here) on line 19

add: Please call directly and refer to the CascadiaFest rate when booking.